### PR TITLE
Known issue for NFD

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@mffiedler PTAL. Thanks!

**Review by mffiedler — DISMISSED** [review]
LGTM

**Comment by mffiedler** [comment]
@wabouhamad PTAL

**Comment by wabouhamad** [comment]
Had a chat with @zvonkok.  For Fresh installs on 4.6.1, we could install the release-4.5 github branch of the NFD operator and deploy it as a workaround.  I propose this updated text:
`The NVIDIA GPU Operator and Node Feature Discovery (NFD) Operator are not available on a fresh installation of {product-title} 4.6.1. To use NFD in version 4.6.1, you must install {product-title} 4.5 and upgrade the cluster to version 4.6.1. Another workaround on a fresh installation of {product-title} 4.6.1 is to install Node Feature Discovery (NFD) Operator from Github repo from CLI:  git clone --branch release-4.5 https://github.com/openshift/cluster-nfd-operator.git; cd cluster-nfd-operator; PULLPOLICY=Always make deploy.
The Operators will be available for fresh cluster installations in a future 4.6.z release. `

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2158,6 +2158,33 @@ In the table below, features are marked with the following statuses:
...
+[source,terminal]
+----
+$ cd cluster-nfd-operator
+----
++
+. Set the pull policy for the repository to always deploy the NFD Operator, so it is deployed to your running cluster:
++
+[source,terminal]
+----
+$ PULLPOLICY=Always make deploy
```
Comment (wking) [inline release_notes/ocp-4-6-release-notes.adoc:2183]: You need to quote this:

```console
$ PULLPOLICY=Always make deploy
make: *** No rule to make target `deploy'.  Stop.
```

**Comment by codyhoag** [comment]
@wabouhamad can you review the additions I made. I tried to add a bit more info with your alternative workflow, but would like to confirm it's accurate. Thanks!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (wking) [reply release_notes/ocp-4-6-release-notes.adoc:2183]: oops, you are expecting to call `make`.  Sorry for the noise

Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:2183]: @wking like this?
```
$ PULLPOLICY="Always make deploy"
```

Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:2183]: Your second comment just rendered for me :thinking: 

Is the original one good then?
```
$ PULLPOLICY=Always make deploy
```

Comment (wabouhamad) [reply release_notes/ocp-4-6-release-notes.adoc:2183]: @codyhoag your updated version LGTM.  thank you

**Comment by wabouhamad** [comment]
@codyhoag LGTM

**Review by sfortner-RH — APPROVED** [review]
LGMT!

**Review by bmcelvee — APPROVED** [review]

**Pull request merged**